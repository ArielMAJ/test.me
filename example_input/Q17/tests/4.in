5
1 3
-1 0 2
3 1
-1
0
2
1 1
5
2 2
1 2
2 1
3 3
.5 .5 .5
.5 .5 .5
.5 .5 .5