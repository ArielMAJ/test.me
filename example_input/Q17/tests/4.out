0.33
0.33
5.00
1.50
0.50
